.circleci/config.yml

```diff
@@ -210,7 +210,7 @@ commands:
       - install_github_bot_deps
       - run:
           name: Report size of RNTester.app (analysis-bot)
-          command: GITHUB_TOKEN="$PUBLIC_ANALYSISBOT_GITHUB_TOKEN_A""$PUBLIC_ANALYSISBOT_GITHUB_TOKEN_B" scripts/circleci/report-bundle-size.sh << parameters.platform >>
+          command: GITHUB_TOKEN="$PUBLIC_ANALYSISBOT_GITHUB_TOKEN_A""$PUBLIC_ANALYSISBOT_GITHUB_TOKEN_B" scripts/circleci/report-bundle-size.sh << parameters.platform >> || true
```

Comment:
Ohoo I see.
Is it possible for us to check the existence of `CIRCLE_PULL_REQUEST` in `report-bundle-size.sh` and log out that we're skipping if it's unset and no-op there?

Reply:
That's what it's doing already though. It is telling us that it's missing the PR number and exits early. It doesn't do any unnecessary work. The problem is that we're treating it as an error and fail the whole pipeline. I don't think it's necessary to block a PR because such errors are not actionable. It could have been a network issue, or Firebase could be down as well.

Reply:
So this error:
`Missing GITHUB_PR_NUMBER. Example: 4687. PR feedback cannot be provided on GitHub without a valid pull request number.`
is coming from here: https://github.com/facebook/react-native/blob/1465c8f3874cdee8c325ab4a4916fda0b3e43bdb/bots/make-comment.js#L79

Which is called from here:
https://github.com/facebook/react-native/blob/1465c8f3874cdee8c325ab4a4916fda0b3e43bdb/bots/report-bundle-size.js#L112

which happens after we've read all the file sizes which we could maybe skip.

Reply:
Ah, you're right, of course! I assumed that it was checked, but we only check `GITHUB_REF` and `GITHUB_SHA`. Good call.

Reply:
Oh whoops, didn't see this comment --I think this is still a worthwhile change, I can extract the validation and update the `post-artifacts-link` to get this off your plate

Reply:
Thanks, but I don't think it would be right to break `post-artifacts-link`. I didn't know `make-comment` was used elsewhere. It should be fixed now.